content/en/runtime.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Runtime
+status: Feedback Appreciated
+category: concept
+tags: ["application", "", ""]
+---
+
+## What it is
+
+A runtime, in general, executes a piece of software.
+It is an [abstraction](/abstraction/) of the underlying operating system that translates the program's commands into respective actions for the operating system. 
+
+In the context of [cloud native](/cloud-native-apps/), _runtime_ generally refers to container runtime. 
+A container runtime specifically implements the [Open Container Initiative](https://opencontainers.org/) specification to ensure consistent handling around the ecosystem. 
+
+## Problem it addresses
+
+Without the abstraction of a container runtime, the application would have to deal with all the mechanics of each operating system, increasing the complexity of running the app. 
+
+## How it helps
+Container runtimes are a necessary component of container orchestrators such as Kubernetes. 
+They handle the container lifecycle, which does mainly two things.
```

Comment:
I accepted your suggestion, @iamNoah1, but I changed the 2nd part of this sentence. You list three things. Is that what container lifecycle is mainly about? Can we say three?